gitpod-io/website
# feat: performance improvement w/ lottie

## Description

We saved loading a whole big chunk JS files of Lottie 😄 

## Related Issue(s)



Fixes #2135

## How to test

Open Below Pages & See 👀 

Affected Pages:

- [`/`](https://deploy-preview-3257--gitpod-kumquat.netlify.app/)
- [`/vs/jetbrains-space`](https://deploy-preview-3257--gitpod-kumquat.netlify.app/vs/jetbrains-space)
- [`/for/enterprise`](https://deploy-preview-3257--gitpod-kumquat.netlify.app/for/enterprise)
- [`/for/education`](https://deploy-preview-3257--gitpod-kumquat.netlify.app/for/education)
- [`/for/startups`](https://deploy-preview-3257--gitpod-kumquat.netlify.app/for/startups)


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3257"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
